src/asm.rs

```diff
@@ -119,7 +119,6 @@ pub fn dump_range(
         used_labels(stmts)
     };
 
-    let mut empty_line = false;
```

Comment:
Looking at your screenshots - yea, they seem redundant - more meaningful on Linux though. Not sure yet how I feel about removing them, will poke around a bit more tomorrow, maybe dropping them is fine.

Reply:
Ok, would you feel better if it was a flag? Not sure how to name it though.

Reply:
>>   So LBB can be breaking code in some logical blocks, while Ltmp looks like something that serves some other purpose.

> Yep, that about what I expected

I'm okay with stripping blank lines from `Ltmp` without extra flags, but I think empty lines from stripped `LBB`  labels can be useful - let's see how the stripping experiment goes.

Reply:
Can you send a screenshot of what the line stripping looks like on Linux? (with and without stripping), so I have some idea on what's going on and what I changed.

Reply:
![1](https://user-images.githubusercontent.com/66211/203086020-a67fa965-e24b-4bae-bc3b-ed2976372116.png)
![2](https://user-images.githubusercontent.com/66211/203086024-7ad5269a-79df-4c3a-97a6-534ecb2539df.png)

Reply:
Actually, I just added back stripping except for temp labels and it work exactly like I want, so I'll do that. 
Just need to refactor it a bit.